### DISCUSSION TITLE
Another inpainting question

### DISCUSSION THREAD

**Discussion by QuickscopingFTW** [nagadomi/nunif#680] [General]
After the recent update the inpainting quality seems to be a fair bit better at higher strengths it doesnt have the hard cutoff where it goes over the training data strength anymore its more gradual, are you planning to ever make/support a model with higher strength than 5 in the training?
Also enjoy your holiday/break you are taking soon

**Comment by nagadomi** [comment]
Recent updates are mainly focused on speed, but if you are referring to monobw_inpaint, it may be because the inpaint mask can now be generated reasonable results even at higher 3D strength.

For inpainting at higher strength, it is not only about the training setup. Larger holes are inherently more difficult, so the outcome is uncertain. For images, the current approach can already handle it to some extent. However, for video, temporal inconsistency will occur.
I think the need for higher 3D strength comes from a desire to view the scene from a closer position. Therefore, I think being able to control the camera position with 3D/4D Gaussian Splatting would likely lead to better results and higher satisfaction.

**Reply by QuickscopingFTW** [reply]
yea I do think another method other than the current way inpainting works might have to be used because albeit as great as the current conversion is the most parallax you can reasonably get with ok quality is about the same as the difference between your left and right eye looking at your finger at the furthest you can hold it, so currently anything that is closer than that distance feels wrong/way too big because there isn't enough parallax, like if you bring your finger 1/3 close to you your eyes can already see right around it to whatever is behind, this isn't really possible at the moment, I hope some future tech makes this easier to implement.
